tests/test_cli.py

```diff
@@ -61,7 +61,16 @@ class CommandLineTestCase(unittest.TestCase):
     def setUpClass(cls):
         super(CommandLineTestCase, cls).setUpClass()
 
-        cls.wd = build_temp_workspace({
+        # Check system's UTF-8 availability, because without it
+        # using UTF-8 paths will break
```

Comment:
I propose:
```suggestion
        # using UTF-8 paths like 'éçäγλνπ¥' will break on Python ⩽ 3.6
```

Reply:
I like how you're using a unicode character in the `<= 3.6` in the comment, where we blame python's unicode support - because we can! :)